pkg/configprovider/globprovider/testdata/mergefunc/a_srcconf.yaml

```diff
@@ -0,0 +1,6 @@
+extensions:
+  sumologic:
+    childKey: "value"
+    collector_fields:
```

Comment:
I wanted to clarify one last thing - if `collector_fields` was renamed to `collector_fields_1`,  my understanding is that the tags would be appended and not replaced. Could we have a test demonstrating this as well?

Reply:
Done

Reply:
Added collector_fields1 along with collector_fields and test will check collector_fields1 will have append behaviour 
 always.